ReceiveMessage/Scripts/sendMessage.js

```diff
@@ -1,17 +1,22 @@
 import MongoConnect from './db'
 import ChatbotMessage from '../models/ChatbotMessage'
+import fixedMessages from '../fixedMessages'
 
 const formResponse = async function (state, message) {
     const p = state.currMessage
+    // if (p == null) {
+    //     state.currMessage = '6022178429efc055c8e74e50'
+    // }
     if (p == null) {
-        state.currMessage = '6022178429efc055c8e74e50'
+        state.currMessage = fixedMessages.get('welcome')
     }
     await MongoConnect()
     const prev = await ChatbotMessage.findById(p)
     const map = prev.nextMessages
     let nextChatbotMessageId = map.get(map.size > 1 ? message : 'default')
     if (nextChatbotMessageId == null) {
-        nextChatbotMessageId = '6022178429efc055c8e74e50' //change this to whatever error message we want to send (for now it is welcome message)
+        // nextChatbotMessageId = '6022178429efc055c8e74e50'
+        nextChatbotMessageId = fixedMessages.get('error message')
```

Comment:
We don't want to set the state to the error message. The point of the error message is to notify the user that they typed something wrong, rather than actually progress them through the module.

Here, it would probably be more fitting to just return the error message as the message to be loaded into the response, bypassing modification of the `currMessage` property of the userState.